index.html

```diff
@@ -95,37 +95,50 @@ <h2>Tonic</h2>
-                  </ul>
-                  <li class="about-lang-menu"><a href="#"><button><h3>Frameworks</h3><img src="Images\about_side_arrow.svg" alt="" width="8px" height="14px"></button></a></li>
-                  <li class="about-lang-menu"><a href="#"><button><h3>Skills</h3><img src="Images\about_side_arrow.svg" alt="" width="8px" height="14px"></button></a></li>
-               </ul>
-         </div>
-        </section>
     </section>
+    <section class="ams">
+      <h2>About Myself</h2>
+      <p>
+         Hello I'm a software developer! I can help you build a product, feature or website
+         Look through some of my work and experiance! If you like what you see and have a project you need coded,
+         don't hestiate to contact me.
+      </p>
+      <h3 class="letconnect_text"><a href="https://github.com/abiy006">LET'S CONNECT</a></h3>
+      <div class="letconnect_icons">
+         <ul>
+            <li><a href="#"><img src="Images\Vector4.svg" alt="" width="17.99px" height="20px"></a></li>
+            <li><a href="#"><img src="Images\Vector2.svg" alt="" width="20px" height="20px"></a></li>
+            <li><a href="#"><img src="Images\Vector5.svg" alt="" width="14.17px" height="20px"></a></li>
+            <li><a href="#"><img src="Images\Vector1.svg" alt="" width="19.99px" height="19.96px"></a></li>
+            <li><a href="#"><img src="Images\Vector3.svg" alt="" width="20px" height="16px"></a></li>
+         </ul>
+      </div>
+      <button>Get my resume</button>
+   <div class="About-languages">
+         <ul>
+            <li class="about-lang-menu"><a href="#"><button><h3>Languages</h3><img src="Images\about_down_arrow.svg" alt="" width="14px" height="8px"></button></a></li>
+            <ul>
+               <li><a href="#"><button><img src="Images\about_javascript2.svg" alt="" width="48px" height="48px"><h3>JavaScript</h3></button></a></li>
+               <li><a href="#"><button><img src="Images\about_html2.svg" alt="" width="48px" height="48px"><h3>HTML</h3></button></a></li>
+               <li><a href="#"><button><img src="Images\about_css2.svg" alt="" width="48px" height="48px"><h3>CSS</h3></button></a></li>
+            </ul>
+            <li class="about-lang-menu"><a href="#"><button><h3>Frameworks</h3><img src="Images\about_side_arrow.svg" alt="" width="8px" height="14px"></button></a></li>
+            <li class="about-lang-menu"><a href="#"><button><h3>Skills</h3><img src="Images\about_side_arrow.svg" alt="" width="8px" height="14px"></button></a></li>
+         </ul>
+   </div>
+  </section>
+    <section class="contact_me_mobile">
+      <h2>Contact me</h2>
+      <p>If you have an application you are intersted in developing, a feature that you need built or a project that
+         needs coding. I'd love to help with it</p>
+      <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] You should use a form_id here not the text {form_id}. You can check this [link](https://formspree.io/html/) and kindly create a form id and use it in your contact form. :+1: